### PR TITLE
Add pop up

### PR DESCRIPTION
**Pull request by BHObserver**
In this branch

- **Added Pop up feature on the project section to see more details about the project**
- **Refactored the whole work section and made it dynamic**
- **Integrated the pop up with the projects object**

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @BHObserver @dd-obua 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't show contribution from your coding partner as this is a paired programming activity your partner is to contribute to this project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -10,27 +10,68 @@
...
+                    <!-- <h3>Keeping track of hundreds of
+                        components</h3> -->
+                    <!-- <ul class="tech-stack">
+                        <li class="tag">Codekit</li>
+                        <li class="tag">Github</li>
+                        <li class="tag">JavaScript</li>
+                        <li class="tag">Bootstrap</li> 
+                        <li class="tag">Terminal</li> 
+                        <li class="tag">Codepen</li> 
+                    </ul> -->
```
Comment (usorfaitheloho) [inline index.html]: Kindly remove all commented lines of code from your project so as to have a clean code that is easily maintainable 😄

File: index.html [hunk 2]
```diff
@@ -152,9 +193,9 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
                         </div>
                     </div>
                 </div>
-            </div>
+            </div> -->
```
Comment (usorfaitheloho) [inline index.html]: Kindly remove all commented lines of code from your project so as to have a clean code that is easily maintainable 😄

File: main.js [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+  {
+    name: 'Multi-Post Stories <br> Gain+Glory',
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.<br><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '<img src="assets/Snapshoot Portfolio.jpg" class="img-pop-up" alt="portfolio-image">',
+    techs: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    live: 'http://bh-observer.me/projects/colorgame/index.html',
+    source: 'https://github.com/bhobserver/bhobserver.github.io/tree/main/projects/colorgame',
+  },
+];
+
```
Comment (usorfaitheloho) [inline main.js:87]: I'd kindly suggest that you add some margin-bottom to the desktop view of your popup window to enale it have a better look and so it is close to the design given 😄 

Your design :
image

Figma design :

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @BHObserver  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Implemented the Popup screen ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] As requested by the previous reviewer in this [review link](https://github.com/BHObserver/my-portfolio/pull/7#discussion_r1273243169). You have not implemented adding the required amount of spacing to the bottom of the popup window as shown in the figma image below. I kindly suggest you make sure to add this change before resubmitting for a re-review 👍 ✨ 

mb

File: index.html (hunk 1 above)
Comment (Efiamotu-1) [inline index.html]: [OPTIONAL]
- [ ] It is unprofessional to leave pieces of commented code in your codebase. I kindly suggest you remove them to keep your code base as clean as possible 👍 ✨

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] Please kindly remove the commented line of code visible on `line 165` in your `index.html` file  
- [ ] It is unprofessional to leave pieces of commented code in your codebase. I kindly suggest you remove them to keep your code base as clean as possible 👍 ✨

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @BHObserver , 



Great work on making the changes requested by a previous reviewer 👏 Your Code is Awesome! 💪
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @BHObserver 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**